MapboxNavigation/RouteViewController.swift

```diff
@@ -9,6 +9,8 @@ public class NavigationPulleyViewController: PulleyViewController {}
 
 public protocol RouteViewControllerDelegate {
     func routeViewControllerDidCancelNavigation(_:RouteViewController)
+    func routeViewController(_ routeViewController: RouteViewController, willChange route: Route)
```

Comment:
Actually, we don't need these now when the simulated route feature resides in the same framework.